inspect_byproducts.py

```diff
@@ -68,54 +65,58 @@ def inspect_byproducts(link, std, operator, inputstring):
 
     <Returns>
       Integer
-        0 - if the inspection is successful
-        1 - if the inspection is unsuccessful
-        2 - if the user supplied arguments are invalid
-
-
+      0 - True
+      1 - False
+      2 - Wrong input
+      3 - IOError (Link file is non existent/ path to the link file is invalid
 
     """
+
     if not os.path.exists(link):
-        print("The path to the link file is invalid")
-        return 2
+        print("The path to the link file is invalid.")
+        return 3
 
-    else:
-      l = link_import.read_from_file(link)
+    imported_link = link_import.read_from_file(link)
 
-      try:
-        std_out_err = l.byproducts[std]
-      except KeyError:
-        raise KeyError("The field corresponding to " + std + " in the link "
-                                                             "file is empty.")
+    try:
+      std_out_err = imported_link.byproducts[std]
+    except KeyError:
+      raise KeyError("The field corresponding to " + std + " in the link"
+                        " file is empty.")
```

Comment:
Choose either double indentation (preferred) or aligned indentation for line continuation